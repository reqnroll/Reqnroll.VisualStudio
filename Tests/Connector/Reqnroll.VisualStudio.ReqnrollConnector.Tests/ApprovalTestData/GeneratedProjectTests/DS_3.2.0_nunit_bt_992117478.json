{
  "GeneratorOptions": {
    "FeatureFileCount": 1,
    "ScenarioPerFeatureFileCount": 1,
    "NewProjectFormat": true,
    "TargetFramework": "net8.0",
    "ReqnrollPackageVersion": "3.2.0"
  }
}
