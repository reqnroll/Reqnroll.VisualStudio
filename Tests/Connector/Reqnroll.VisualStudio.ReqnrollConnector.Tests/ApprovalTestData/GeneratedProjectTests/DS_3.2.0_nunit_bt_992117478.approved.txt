{
  "stepDefinitions": [],
  "hooks": [],
  "sourceFiles": {},
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "Reqnroll.dll",
    "SFFileVersion": "3.0.0",
    "SFProductVersion": "3.2.0+31832046767fc043c9e10d0cc23591c819071bc1",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETCoreApp,Version=v8.0",
    "TypeNames": "0"
  }
}